src/index.js

```diff
@@ -1,38 +1,81 @@
 import './style.css';
+import {
+  Task,
+  addTask,
+  removeTask,
+  checkLocalStorage,
+  editTask,
+} from './module/Function.js';
+import * as Elements from './module/constElements.js';
 
-const listSection = [
-  {
-    index: 0,
-    description: 'wash the dish',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'complete to do list project',
-    completed: false,
-  },
-];
-
-const displayContent = () => {
-  const tasks = document.querySelector('#listSection');
-  listSection.forEach((element) => {
-    tasks.innerHTML += `
-          <div class="border-bottom  m-0 px-3 py-0 d-flex align-items-center justify-content-between">
-            <div class="form-check mb-0 d-flex align-items-center justify-content-start">
-              <input class="form-check-input border" type="checkbox" value="" id="flexCheckDefault">
-              <label class="form-check-label p-3 m-0" for="flexCheckDefault">
-                <span class="h5 m-0 p-0">${element.description}</span>
-              </label>
-            </div>
-            <button id="three-dots" class="h5 btn m-0 icon">
-              <i class="fa-solid fa-ellipsis-vertical"></i>
-            </button>
-            <button id="trash" class="h5 btn m-0 icon hide">
-              <i class="fa-solid fa-trash-can"></i>
-            </button>
-          </div>
-        `;
+/* eslint-disable */
+/* eslint-enable */
+
+Elements.submitInput.addEventListener('click', addTask);
+
+Elements.taskInput.addEventListener('keypress', (event) => {
+  if (event.keyCode === 13) {
+    event.preventDefault();
+    addTask();
+    Elements.taskInput.value = '';
+  }
+});
+
+Elements.refreshTask.addEventListener('click', (e) => {
+  e.preventDefault();
+  Task.TaskObject = [];
+  localStorage.setItem('TASKS_LIST', JSON.stringify(Task.TaskObject));
+  checkLocalStorage();
+});
+
+Elements.taskList.addEventListener('click', (e) => {
+  e.stopPropagation();
+
+  [...Elements.taskList.children].forEach((item, index) => {
```

Comment:
- [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5). Also, deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. Kindly ensure to improve upon this aspect.

**Index mismatch:**
![image](https://user-images.githubusercontent.com/12745474/161207373-7f5c2351-4a62-42e2-8b7e-03c64f99b05a.png)

Reply:
Thank you @rloterh for your review
For this part the description property is used to match the value
<img width="596" alt="Screenshot 2022-04-01 at 7 50 11 AM" src="https://user-images.githubusercontent.com/60798978/161219617-ddc4b553-7b20-4902-8565-efb035937e8f.png">

I'm using the code you highlighted to change the color of the taskbar when clicked to yellow and also enable the delete functionality

Reply:
@Estherstarr 
This requirement simply means that the first entry should have an index of 1, and the second an index of 2, and so on. Assuming the first element is deleted, then the second element should replace the first element and then it should have an index of 1. Cheers!

Reply:
@rloterh how do I enable this feature. Can you give me some guidelines

Reply:
@Estherstarr 
We have had a discussion on the same via slack